R/estimate.R

```diff
@@ -0,0 +1,125 @@
+#'  Source code for the ESTIMATE algorithm:
+#'  Estimate of Stromal and Immune Cells in Malignant Tumor Tissues from Expression Data
+#'  (https://doi.org/10.1038/ncomms3612)
+#'
+#'  Source code adapted from: http://download.r-forge.r-project.org/bin/windows/contrib/3.3/estimate_1.0.13.zip
```

Comment:
I think you should include the license and the original copyright information here

Reply:
I could map the estimate scores, but it would be a 1:1 correspondance since we have 4 scores (immnue, stromal, estimate and tumor purity). I don't think it would have that much sense. I could just renamre the columns and forget about it. 

The `map_cell_type` function can be used either with the `deconvolute` or with the `deconovlute_mouse` function ,and not with the method specific ones, es `deconvolute_cibersort`, since the output of those functions are not uniformed
I would avoid doing the remapping because it seems excessive

Reply:
But Estimate is not the only GPL 2.0 licensed method. TIMER, Abis and DCQ all have GPL 2.0 licenses. And if I'm not mistaken with TIMER you included directly the code rather than a package

Reply:
as discussed on slack ...